### ISSUE TITLE
Uncaught ReferenceError on details view

### ISSUE THREAD

**Issue by lovtsovaik** [jmix-framework/jmix#1831]
Jmix version: 2.0.999-SNAPSHOT

**Test case**

- Create a simple entity with one string field
- Create CRUD views for it
- Run project
- Open editor
- Close editor

AR: UI errors 
on open:
```
Exception is thrown during JavaScript execution. Stacktrace will be dumped separately.

The error has occurred in the JS code: '$0, return (function() { window.addEventListener('beforeunload', jmixBeforeUnloadListener, {capture: true})}).apply($0)'

Uncaught ReferenceError: jmixBeforeUnloadListener is not defined (http://localhost:8080/VAADIN/build/FlowClient-e0ae8105.js:1:7941)
```
on close:
```
Exception is thrown during JavaScript execution. Stacktrace will be dumped separately.

The error has occurred in the JS code: '$0, return (function() { window.removeEventListener('beforeunload', jmixBeforeUnloadListener, {capture: true})}).apply($0)'

Uncaught ReferenceError: jmixBeforeUnloadListener is not defined (http://localhost:8080/VAADIN/build/FlowClient-e0ae8105.js:1:7941)
```

**Comment by glebfox** [comment]
Check that `frontend/index.html` contains 

```
<script type="text/javascript">
  jmixBeforeUnloadListener = (event) => {
    event.preventDefault();
    return (event.returnValue = "");
  };
</script>
```

The code above is included to the project template since 2.0 and must be added by studio during migration process, see: https://youtrack.haulmont.com/issue/JST-3776/Alter-frontendindexhtml-when-migrating-to-20

**Comment by lovtsovaik** [comment]
yes, problem was in frontend/index.html
